shared/build.gradle.kts

```diff
@@ -41,14 +41,21 @@ kotlin {
     sourceSets {
         val commonMain by getting {
             dependencies {
-                //put your multiplatform dependencies here
+                // Koin
+                implementation(Dependencies.Koin.CORE)
             }
         }
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test"))
             }
         }
+        val androidMain by getting {
+            dependencies {
+                implementation(Dependencies.Koin.ANDROID)
+                implementation(Dependencies.Koin.COMPOSE)
+            }
```

Comment:
I think we need them for the shared module in androidMain. We set them in the "android" module instead?

Reply:
@luongvo did you mean `I don't think we need them for the shared module in androidMain. We set them in the "android" module instead?` ?

Reply:
@kaungkhantsoe Ah yes, my bad. `I don't think`... 🙏